pkgs/servers/web-apps/plausible/default.nix

```diff
@@ -72,7 +62,8 @@ beamPackages.mixRelease {
   meta = with lib; {
     license = licenses.agpl3Plus;
     homepage = "https://plausible.io/";
-    description = " Simple, open-source, lightweight (< 1 KB) and privacy-friendly web analytics alternative to Google Analytics.";
+    changelog = "https://github.com/plausible/analytics/blob/${src.rev}/CHANGELOG.md";
+    description = " Simple, open-source, lightweight (< 1 KB) and privacy-friendly web analytics alternative to Google Analytics";
     maintainers = with maintainers; [ ];
```

Comment:
@softinio Should I add you?

Reply:
on the fence about it as plausible stack (js/elixir) not my strong skill :) on the other hand having me as maintainer better than having no one :)

Reply:
The most important consequence is that you'd be pinged for any change, so you can review it.

Reply:
go for it then. I'll give it a shot to help. [this](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix#L16062-L16067).

Thanks.